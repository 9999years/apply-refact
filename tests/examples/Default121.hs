import qualified Control.Monad as CM 
yes = flip mapM