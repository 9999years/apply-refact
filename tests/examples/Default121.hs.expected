import qualified Control.Monad as CM
yes = Control.Monad.forM
